NDB.Covid19/NDB.Covid19.iOS/NDB.Covid19.iOS.csproj

```diff
@@ -562,12 +562,24 @@
     <ImageAsset Include="Assets.xcassets\MenuIcon.imageset\hamburgermenuicon%403x.png">
       <Visible>false</Visible>
     </ImageAsset>
-    <ImageAsset Include="Assets.xcassets\FHISmittestoppLogo.imageset\Contents.json" />
-    <ImageAsset Include="Assets.xcassets\FHISmittestoppLogo.imageset\FHI_Smittestopp_App_Logo_Aktiv_Inaktiv%401x.png" />
-    <ImageAsset Include="Assets.xcassets\FHISmittestoppLogo.imageset\FHI_Smittestopp_App_Logo_Aktiv_Inaktiv%402x.png" />
-    <ImageAsset Include="Assets.xcassets\FHISmittestoppLogo.imageset\FHI_Smittestopp_App_Logo_Aktiv_Inaktiv%403x.png" />
-    <ImageAsset Include="Resources\Assets.xcassets\Contents.json" />
-    <ImageAsset Include="Assets.xcassets\AppIconNO.appiconset\iTunesArtwork%402x.png" />
+    <ImageAsset Include="Assets.xcassets\FHISmittestoppLogo.imageset\Contents.json">
+      <Visible>false</Visible>
```

Comment:
Why have this and the ones below here been changed ?   The property added that is.

Reply:
This is likely anv accidental change. It should be reverted